ts/parts/Navigator.ts

```diff
@@ -2234,6 +2283,17 @@ Navigator.prototype = {
                     baseNavigatorOptions
                 );
 
+                // Once nav series type is resolved, pick correct pointRange
+                mergedNavSeriesOptions.pointRange = pick(
+                    // Stricte set pointRange in options
+                    userNavOptions.pointRange,
+                    baseNavigatorOptions.pointRange,
+                    // Fallback to default values, e.g. `null` for column
+                    (defaultOptions.plotOptions as any)[
```

Comment:
@bre1470 - I know I introduce here new `as any`, but the same happens in other files where we need to reference `defaultOptions.plotOptions`. I assume this will be picked altogether with other changes?

Reply:
I believe this should not be necessary inside the pick function anymore. @jon-a-nygaard made it type safe.

Reply:
Oh I see now, that it is used with an indexer. `any` is okay in this case.

Reply:
Ok, thanks! Sorry, should have highlight more lines to make it more readable..